### DISCUSSION TITLE
Weekly demand forecast

### DISCUSSION THREAD

**Discussion by JATDALTON** [frePPLe/frepple#656] [Q&A]
I am new to frePPLe and working to get a proof of concept going. 

I currently have one item with several locations. 

 My sales orders are loaded and have dates that are at the day level. When I set the forecast parameters to monthly  I get what looks to be a reasonable first step in the demand planning process, however when I set the parameters to weekly the demand forecast is materially below where I would expect it to be. No changes to the underlying data have been made, just the change in parameter.  

The images below show the resulting forecasts, the first monthly, the second weekly. Any suggestions as to what may be causing this discrepancy?

Edit to add: I have also reduced down to one item and one location to simplify and then grow the data set, the result is the same.  Any help is appreciated. I can provide additional data as well if needed.

![image](https://github.com/frePPLe/frepple/assets/170448333/b37d8290-811c-41e3-8390-1fd954fc0066)

![image](https://github.com/frePPLe/frepple/assets/170448333/e2a22357-4082-4512-895f-a875f5e28e64)
[parameters.xlsx](https://github.com/frePPLe/frepple/files/15392481/parameters.xlsx)

**Comment by jdetaeye** [comment]
You changed the "forecast.calendar" parameter, but all other forecasting related parameters are left at the value for the monthly buckets.

You need to load the parameters that go with your bucket size. This is easily done in the "admin / execute" screen with the "load a dataset" task.

![image](https://github.com/frePPLe/frepple/assets/5219207/a2a8a119-1a49-40f8-b16f-c6eb9caa3767)

Note: In the upcoming 9.0 version, this is simplified. By default we pick up the parameter value suitable for your bucket size. Loading the new parameters is then no longer needed.

**Comment by hichamlahlou** [comment]
I also suspect you have no demand history for the last 2 weeks when running in weekly buckets (I can see a gap between the end of the green line and the beginning of the red line). These are considered as zero sales by the forecast engine and can significantly decrease your computed forecast values.

**Comment by JATDALTON** [comment]
Thanks for the tips, I believe that the main culprit was the forecast date setting, I was trying to get a clean cutoff for data but was not adjusting that date. I look to be all set now. Thank you
